src/zkbob/utils/PriorityQueue.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.13;
+
+/**
+ * @dev The structure that stores all information about the pending commitment.
+ * @param commitment commitment value to be added in the Merkle Tree.
+ * @param prover address of the prover that submitted the commitment.
+ * @param fee fee reserved for the prover who will submit the tree update proof.
+ * @param timestamp commitment timestamp.
+ */
+struct PendingCommitment {
+    uint256 commitment;
+    address prover;
+    uint64 fee;
+    uint64 timestamp;
+}
+
+/// @author Matter Labs
+/// @dev The library provides the API to interact with the priority queue container
+/// @dev Order of processing operations from queue - FIFO (Fist in - first out)
+library PriorityQueue {
```

Comment:
Why is it called a priority queue if there is no priority? :)
Just a regular queue / fifo?

Reply:
Renamed it in https://github.com/zkBob/zkbob-contracts/pull/97/commits/6d16f95b0ac5a7bbefdf566e022423d8e7062308.